### ISSUE TITLE
Naming of executables

### ISSUE THREAD

**Issue by HadrienG** [SGBC/metlab_tools#3]
We need to get the binaries to contain the name of the target OS

we could either add a conditional in `.travis.yml` or in `./compile.sh` or in `deploy.sh` 😁

**Comment by HadrienG** [comment]
After reflection, I vote for `compile.sh`:

The compilation stays the same, we change the name during `mv`

```shell
mkdir ../samtools
mv samtools ../samtools/
if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
    tar czf ../samtools-darwin.tar.gz ../samtools
else tar czf ../samtools-linux.tar.gz ../samtools
fi
```

**Comment by HadrienG** [comment]
fixed with c087aa7b28f55d94ca5f89306cc901dca8b46aae
